blocks/library/list/index.js

```diff
@@ -7,6 +7,34 @@ import Editable from 'components/editable';
 
 const { children, prop } = hpq;
 
+function activeEditorExecCommand( command ) {
+	return () => {
+		const ed = tinymce.activeEditor;
```

Comment:
The problem with that is we're moving code that likely no-one else is going to use into a common object purely to work around access issues. That doesn't appeal to me greatly either.